src/HostManager.cpp

```diff
@@ -128,11 +128,9 @@ void HostManager::postIrcMessage(QString a, QString b, QString c)
 // send out the events to the other hosts in a predictable and consistent order
 // and so that no one host gets an unfair advantage when emitting events. The
 // sending profile host does NOT get the event!
-// Note: Optional forceGlobal allows passing a null pointer as pHost, and will raise
-// an event for all profiles.
-void HostManager::postInterHostEvent(const Host* pHost, const TEvent& event, const bool forceGlobal)
+void HostManager::postInterHostEvent(const Host* pHost, const TEvent& event)
```

Comment:
The changes here and in `src/HostManager.cpp` do not seem to need/want reversion?